lib/models/emailAction.js

```diff
@@ -52,8 +52,13 @@ function SendMail(action, event, callback) {
 
         transporter.sendMail(mailOptions, function(err, info) {
             logger.debug('emailAction.SendMail %j %j %j', mailOptions, err, info);
+            // Not a HTTP request, must count outgoingTransaction
```

Comment:
NTC, aready done

(I was looking to an older version of the code when I wrote the comment, sorry...)